examples/dh-5-example/build.gradle

```diff
@@ -0,0 +1,24 @@
+buildscript {

```

Comment:
This buildscript only works on the company VPN. It can be simplified to this:

```
plugins {
    id 'net.saliman.properties' version '1.4.6'
    id 'com.marklogic.ml-data-hub' version '5.0.0'
}

task extractZip (type: Copy) {
    from zipTree('input.zip')
    def destDir = System.getProperty("user.dir")
    println("input directory is extracted to: " + destDir + "/input")
    destinationDir = file(destDir)
}
```

Reply:
Changed it to use id 'com.marklogic.ml-data-hub' version '5.0.0'